mgmtfn/mesosplugin/contiv-isolator/netpluginhandler.go

```diff
@@ -0,0 +1,163 @@
+/***
+Copyright 2016 Cisco Systems Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/contiv/netplugin/mgmtfn/mesosplugin/api"
+)
+
+const driverPath = "/run/mesos-netmodules-netplugin.sock"
```

Comment:
It seems like this belongs in a common place shared between netplugin and contiv-isolator